apps/python-sdk/firecrawl/v2/methods/aio/vector_search.py

```
@@ -0,0 +1,159 @@
+        data = response_data.get("data", {}) or {}
+        
+        # Normalize timing data from camelCase to snake_case
+        timing_data = data.get("timing", {})
+        normalized_timing = {}
+        if "queryEmbeddingMs" in timing_data:
+            normalized_timing["query_embedding_ms"] = timing_data["queryEmbeddingMs"]
+        if "vectorSearchMs" in timing_data:
+            normalized_timing["vector_search_ms"] = timing_data["vectorSearchMs"]
+        if "totalMs" in timing_data:
+            normalized_timing["total_ms"] = timing_data["totalMs"]
+        
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)